ngraph/core/include/ngraph/pass/pass_config.hpp

```diff
@@ -1,46 +0,0 @@
-//*****************************************************************************
```

Comment:
Why do you remove this file? It is not a backward compatible change

Reply:
It wasn't used anywhere. But probably you are right. Will keep it under deprecated marcos.

Reply:
With @andrew-zaytsev help we decided to add a note about this class deprecation inside our docs https://github.com/openvinotoolkit/openvino/pull/2593 So this helps us to remove this classes